client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -72,6 +75,31 @@ class BackupsPage extends Component {
 	};
 
 	renderMain() {
+		const { isRewindMissingPlan, siteId, siteSlug } = this.props;
+
+		return (
+			<Main>
+				<DocumentHead title="Backups" />
+				<SidebarNavigation />
+				<QueryRewindState siteId={ siteId } />
+				<QuerySitePurchases siteId={ siteId } />
+				{ isRewindMissingPlan ? (
+					<Card>
```

Comment:
Why not have the `<Card` be part of the BackupUpsell component?

Reply:
My impression is most things will be in Cards eventually based on the Desktop designs, but for now it can be part of the component!